### PR TITLE
Edit concept html elements

### LINKED ISSUE #1735: [Edit] HTML: Elements
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

Input from [comment](https://github.com/Codecademy/docs/pull/533#issuecomment-1026074172).

Make edits to content/html/concepts/elements/elements.md:

> Some elements, like input, img, br, etc. are void elements (aka "empty" elements in the current description, elements that can never have children)
> 
> void elements can optionally be self-closing (ie have a trailing slash) and it's pretty much up to personal preference
> there is also a caveat that making all void elements self-closing improves compatibility with XHTML

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by cslylla**
### Description

- Edit term (concept): content/html/concepts/elements/elements.md
- Incorporate insights from a comment: add void elements and self-closing elements
- Update pseudo code and description, rewrite sentences, reorganize content order
- Closes #1735 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by PetrZajkasWebdesign** [comment]
Hi can I contribute on this one please?

**Comment by SSwiniarski** [comment]
@PetrZajkasWebdesign, this is someone else's PR. If you want something to contribute to, you can find an unclaimed issue on the [issues tab](https://github.com/Codecademy/docs/issues).

**Comment by PetrZajkasWebdesign** [comment]
ok many thanks

File: content/html/concepts/elements/elements.md [hunk 1]
```diff
@@ -16,17 +16,49 @@ An HTML element is a piece of content in an HTML document.
 
 ## Syntax
 
+Elements consist of at least one tag but can have two tags with optional content. The tag name determines how the element will be rendered in the HTML document. Elements can also have [attributes](https://www.codecademy.com/resources/docs/html/attributes).
```
Comment (caupolicandiaz) [inline content/html/concepts/elements/elements.md:19]:
```suggestion
**Elements** are discrete components within an HTML document that generally consist of at least one tag but can have two tags with optional content. The tag name determines how the element will be rendered in the HTML document. Elements can also have [attributes](https://www.codecademy.com/resources/docs/html/attributes).
```

Comment (caupolicandiaz) [reply content/html/concepts/elements/elements.md:19]: I think this should be the introductory text and should move up to line 17. The  "syntax" header should come after this paragraph, and perhaps we could follow it with a sentence or two to preface the details of each of the element variations.

File: content/html/concepts/elements/elements.md [hunk 2]
```diff
@@ -16,17 +16,49 @@ An HTML element is a piece of content in an HTML document.
...
+### Void HTML Element
+
+```pseudo
+<void-element-tag>
+```
+
+Some elements cannot have content; therefore, they do not require a closing tag. These elements are called void or empty elements.
 
-<self-closing-element />
+### Self Closing HTML Element
```
Comment (caupolicandiaz) [inline content/html/concepts/elements/elements.md:37]:
```suggestion
### Self-Closing HTML Element
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cslylla Just added a few edits. Please tag me when you've finished.

File: content/html/concepts/elements/elements.md (hunk 1 above)
Comment (cslylla) [reply content/html/concepts/elements/elements.md]: I updated the text accordingly.

**Comment by cslylla** [comment]
@caupolicandiaz I changed the paragraphs, as you requested, and added more information.

**Review by caupolicandiaz — APPROVED** [review]
@cslylla Looks good, just made one small edit to the language on line 19.

**Comment by SSwiniarski** [comment]
@cslylla congrats on the PR. The edits can be seen here:https://www.codecademy.com/resources/docs/html/elements

**Pull request merged**